File: GestionInterim/app/src/main/java/com/example/gestioninterim/slides/FragmentSlideInterim.java
package com.example.gestioninterim.slides;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.fragment.app.Fragment;

import com.example.gestioninterim.R;


public class FragmentSlideInterim extends Fragment {


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {

        return inflater.inflate(R.layout.fragment_slide_child_interim, container, false);
    }
}

File: GestionInterim/app/src/main/java/com/example/gestioninterim/slides/FragmentSlideInterim.kt
package com.example.gestioninterim.slides

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.example.gestioninterim.R

class FragmentSlideInterim : Fragment() {
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.fragment_slide_child_interim, container, false)
    }
}